docs/Linting_Prettier.md

```diff
@@ -0,0 +1,107 @@
+# About
+
+This file is to help our newer volunteers understand what was implemented and pinpoint appropriate & related resources.
+
+Automated linting & prettier formatting was first introduced with [#49](https://github.com/codefordallas/codefordallas.github.io/pull/49) and has some information there, which will be pulled in here. Additional issues and PRs are [#54](https://github.com/codefordallas/codefordallas.github.io/issues/54), [#57](https://github.com/codefordallas/codefordallas.github.io/pull/57), and [#59](https://github.com/codefordallas/codefordallas.github.io/issues/59).
+
+This specific update is related to [#58](https://github.com/codefordallas/codefordallas.github.io/issues/58).
+
+## Adding The Packages
+
+As per [here](https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file) you would run `npm install <package-name> --save-dev` to save them as [`devDependencies`](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#devdependencies) in your `package.json` file.
+
+Examples might look like:
+
+```
+npm install eslint --save-dev
```

Comment:
I think preferred is that they just run `npm install` instead of installing things individually. Unless we're just trying to explain the concept here

Reply:
this was for concept yes, but we definitely don't want to cause confusion if we can avoid it - I'll look at updating this shortly